packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -436,7 +436,8 @@ export const EVENT_TRIGGERS = {
   MEDIA_QUALITY: 'media:quality',
   MEETINGS_NETWORK_DISCONNECTED: 'network:disconnected',
   MEETINGS_NETWORK_CONNECTED: 'network:connected',
-  MEETING_SELF_LEFT: 'meeting:self:left'
+  MEETING_SELF_LEFT: 'meeting:self:left',
+  NETWORK_QUALITY_JUDGMENT: 'network:quality:judgment'
```

Comment:
can we check with the team , should we need to use the judgment ?

Reply:
I used the word judgment as it has the same type of connotation as a boolean, like a court judgment (innocent or guilty) true or false - because we are basically stating whether the network is acceptable or not, true or false, black or white, etc.  Obviously it can be changed - I thought about using assessment, conclusion, verdict, analysis, etc.  Whatever works for group is fine with me

Reply:
name updated